pandas/tests/io/parser/test_parse_dates.py

```diff
@@ -840,6 +826,8 @@ def test_parse_dates_arrow_engine(all_parsers):
             "b": 1,
         }
     )
+    if parser.engine == "pyarrow":
+        expected["a"] = expected["a"].dt.as_unit("s")
```

Comment:
had to make the same change in my branch.  does this mean the pyarrow csv parser is giving back second unit?  i thought it standardized on "us"